### PR TITLE
feat: add `visudo` spec

### LINKED ISSUE #1528: [visudo] missing spec

### PR DESCRIPTION
**Pull request by victorpatru**
**What kind of change does this PR introduce?**

Pull request for issue (closes #1528) using [visudo's publicly available documentation](https://www.sudo.ws/docs/man/visudo.man/)

_Making my first contribution to Fig's Autocomplete Repo. Happy to make any changes if needed!_

### CODE REVIEW

File: src/visudo.ts [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+    {
+      name: ["--check", "-c"],
+      description: "Checking existing sudoers file for syntax errors",
+    },
+    {
+      name: ["--file", "-f"],
+      description: "Set an alternative sudoers file location",
+      args: {
+        name: "New sudoers location",
+        description: "Specify the alternate sudoers file location",
```
Comment (fedeci) [inline src/visudo.ts:15]:
```suggestion
        description: "Specify the alternate sudoers file location",
        template: "filepaths",
```

File: src/visudo.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    },
+    {
+      name: ["--version", "-V"],
+      description: "Display version and exit",
+    },
+    {
+      name: ["--export", "-x"],
+      description: "Export JSON and write it to output_file",
+      args: {
+        name: "Output file name",
```
Comment (fedeci) [inline src/visudo.ts:38]:
```suggestion
        name: "Output file name",
        template: "filepaths",
        suggestCurrentToken: true,
```

**Review by fedeci — CHANGES_REQUESTED** [review]
Thanks for the spec! I left a couple comments to address before merging this PR.

**Comment by victorpatru** [comment]
Thanks @fedeci. Merged the comments! Lmk if there is anything else required before merging.

**Review by fedeci — APPROVED** [review]
Thanks!

**Pull request merged**